sendgrid/helpers/mail/mail.py

```diff
@@ -200,7 +200,12 @@ def contents(self):
     def add_content(self, content):
         if self._contents is None:
             self._contents = []
-        self._contents.append(content)
+        
+        #Fix for issue-451: text content should be before html content
+        if content._type == "text/plain":
```

Comment:
What will happen here if someone provides only html content?

Reply:
@hugomallinson who reported the [issue](https://github.com/sendgrid/sendgrid-python/issues/451) mentioned that it was only when there is both plain text and HTML content and the HTML content is specified first.

If there's only HTML content, there shouldn't be any error and order wouldn't matter/make a difference